### PR TITLE
feature/publish wizard page if first

### PR DESCRIPTION
**Pull request by czpython**
When creating page through admin, the first page is always published automatically.
This PR adds logic to the page wizard to do the same.

### CODE REVIEW

File: cms/forms/wizards.py [hunk 1]
```diff
@@ -294,6 +294,12 @@ def save(self, **kwargs):
                         }
                         add_plugin(**opts)
 
+        # is it home? publish it right away
+        pages_in_site = Page.objects.filter(site_id=page.site_id)
+
+        if not self.page and pages_in_site.count() == 1:
```
Comment (piotrjakimiak) [inline cms/forms/wizards.py]: I think `not pages_in_site.exclude(pk=page.pk).any()` will have better performance.

Comment (czpython) [inline cms/forms/wizards.py]: @piotrjakimiak Do you mean ``.exists()``?

Comment (piotrjakimiak) [inline cms/forms/wizards.py]: Yes, I mean `exists()`

**Comment by mkoistinen** [comment]
LGTM

**Pull request merged**